src/jacobogart.js

```diff
@@ -1,4 +1,5 @@
-const bars = [
+const data = {
```

Comment:
I assume this file is so that we can access the data locally, up until we receive the API link

Reply:
Yup! We will have to change it from imports to fetch calls once we get the API link.

Reply:
We will definitely update this later on!